company-profiles/picpay.md

```diff
@@ -0,0 +1,43 @@
+# PicPay
```

Comment:
Company name should match the readme so it should be Picpay or adjust in Readme :smile:

Reply:
Of course 😳, I made another commit adjusting the company name